src/content/reference/react/lazy.md

```diff
@@ -16,63 +16,63 @@ const SomeComponent = lazy(load)
 
 ---
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
 ### `lazy(load)` {/*lazy*/}
 
-Call `lazy` outside your components to declare a lazy-loaded React component:
+Panggil fungsi `lazy` di luar komponen apapun untuk mendeklarasikan *lazy-loaded* komponen React:
 
 ```js
 import { lazy } from 'react';
 
 const MarkdownPreview = lazy(() => import('./MarkdownPreview.js'));
 ```
 
-[See more examples below.](#usage)
+[Lihat contoh-contoh lainnya di bawah ini.](#usage)
 
 #### Parameters {/*parameters*/}
```

Comment:
```suggestion
#### Parameter {/*parameters*/}
```

Suggested change for API reference headings, see: https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5769320